dist/common/scripts/scylla_coredump_setup

```diff
@@ -24,6 +24,9 @@ import os
 import sys
 import argparse
```

Comment:
The changelogs says --skip-test was added, but there is no --skip-test in the patch.

I remember we wanted to remove the test from rpm/deb installation, is this done?